src/families/ethereum/EditFeeUnitEthereum.js

```diff
@@ -111,9 +111,7 @@ export default function EditFeeUnitEthereum({
 }
 
 const styles = StyleSheet.create({
-  root: {
-    flex: 1,
-  },
+  root: {},
```

Comment:
If we don't need any styles, drop this 🤷🏼

Reply:
done 👌